config/default.json

```diff
@@ -17,6 +17,9 @@
   "connector": {
     "configFile": "/usr/local/bin/knot-fog-connector/config/default.json"
   },
+  "device": {
+    "configFile": "/usr/local/bin/thingd/config/default.json"
```

Comment:
This file must be in a `.conf` format instead of JSON as I sent to you in the slack channel.

Reply:
Done!